File: src/main/java/it/manuelgozzi/blog/repository/PostMongoReactiveRepository.java
package it.manuelgozzi.blog.repository;

import it.manuelgozzi.blog.model.Post;
import org.springframework.data.repository.reactive.ReactiveCrudRepository;
import org.springframework.stereotype.Repository;

/**
 * @author Manuel Gozzi
 */
@Repository
public interface PostMongoReactiveRepository extends ReactiveCrudRepository<Post, String> {
}

File: src/main/kotlin/it/manuelgozzi/blog/repository/PostMongoReactiveRepository.kt
package it.manuelgozzi.blog.repository

import it.manuelgozzi.blog.model.Post
import org.springframework.data.repository.reactive.ReactiveCrudRepository
import org.springframework.stereotype.Repository

/**
 * @author Manuel Gozzi
 */
@Repository
interface PostMongoReactiveRepository : ReactiveCrudRepository<Post, String>